### PR TITLE
[WIP]Create new libyui-rest-api based test modules for ntp_config_settings

### PR DESCRIPTION
**Pull request by tinawang123**
- Related ticket: https://progress.opensuse.org/issues/133001
- Verification run:  https://openqa.suse.de/tests/11729876

### CODE REVIEW

File: tests/installation/ntp_configuration.pm [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+use Test::Assert ':all';
+
+sub run {
+    my $ntp_settings = $testapi::distri->get_ntp_config();
+    diag '*****' . $ntp_settings;
+    assert_not_matches(qr/suse\.pool\.ntp\.org/, $ntp_settings, "NTP setting is not correct");
```
Comment (mloviska) [inline tests/installation/ntp_configuration.pm:20]: Should the module run for MicroOS as well? If so there is opensuse pool server used.

**Review by jlausuch — COMMENTED** [review]
Ofc, I leave it up to the implementer how to do it, but there is already a function [load_installation_tests](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/main_micro_alp.pm#L82)

which loads the installation tests and handles the boot as well...  The schedule also checks for addons and modules:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/main_micro_alp.pm#L341

Not sure if we need to duplicate things...

File: lib/DistributionProvider.pm [hunk 2]
```diff
@@ -33,7 +33,7 @@ If there is no matched version, then returns Tumbleweed as the default one.
 =cut
 
 sub provide {
-    return Distribution::Sle::15_current->new() if is_sle('>=15-sp3');
+    return Distribution::Sle::15_current->new() if (is_sle('>=15-sp3') || is_sle_micro);
```
Comment (ricardobranco777) [inline lib/DistributionProvider.pm:36]: And Tumbleweed?

File: lib/Installation/NtpConfiguration/NtpConfigurationPage.pm [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    return $self;
+}
+
+sub is_shown {
+    my ($self) = @_;
+    return $self->{txb_ntp_server}->exist();
+}
+
+sub get_ntp_config {
+    my ($self, $ntpserver) = @_;
```
Comment (ricardobranco777) [inline lib/Installation/NtpConfiguration/NtpConfigurationPage.pm:28]: You're not using `$ntpserver`.

**Review by ricardobranco777 — CHANGES_REQUESTED** [review]

File: lib/Installation/NtpConfiguration/NtpConfigurationPage.pm (hunk 3 above)
Comment (tinawang123) [reply lib/Installation/NtpConfiguration/NtpConfigurationPage.pm]: No, I don't use this. Deleted it. Thanks.

**Comment by tinawang123** [comment]
New PR: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17575

**Pull request closed without merging**